watchface

```diff
@@ -211,7 +211,7 @@ function guiMenu {
         elif [ "${action}" = "deploy" ] ; then
             local first=true
             for wf in ${watchface}; do
-                deployface "${wf}" "${first}"
+                deployface "${wf//\"/}" "${first}"
```

Comment:
There should be no quotation marks to remove here.   Since `zenity` is used to create the file list in the `guiMenu` function, and either `dialog` or `whiptail` is used to create the file list in the `textMenu` function, it seems unlikely that both have different behavior on your machine vs. my machine.  I think we need to dig a little deeper into why there are quotation marks in there at all and how they get introduced.

Reply:
`$wf` has double quotes on my machine. This is the part I'm trying to figure out.